### ISSUE TITLE
Performance with large number of indexes

### ISSUE THREAD

**Issue by sdesse** [ElasticHQ/elasticsearch-HQ#508]
### General information

- **OS Platform and Distribution**: Debian 9
- **ElasticHQ Version**: 3.5.0
- **Elasticsearch Version**: 6.8.4
- **Python version (ignore is using docker image)**: Python 3.4.2
- **Browser Vendor and Version (if applicable)**: Any Chrome, FF, Edge

### Issue Description

We have 11k+ indexes and loading the elasticHQ page takes a lot of time. Looking at the logs it seems that it is because it retreives the stats for all the indexes when loading the page.

Would there be a way to not load the indexes, filter the ones that are loaded or to optimize this behavior? 

### Be Patient

Thanks again!

**Comment by clintonelston777** [comment]
3.4 k here and after loading about 30 seconds later docker crash (looks like jvm) and nothing works after that. Can you stop loading everything on the initial screen maybe?

**Comment by dekimsey** [comment]
Agreed, there are a few places where it doesn't work well. Another is the query page which tries to load all the indices into the drop-down.

I think loading the data is fine, but maybe that can happen in the background without affecting the UI (if the data is cached a minute or three I don't think that's huge loss).
